Web/starter_project_barbies/pages/about/index.tsx

```diff
@@ -0,0 +1,20 @@
+import HeroSection from '../../components/about/HeroSection'
```

Comment:
[nit] i think you can use `@/components/...` for your paths (this might apply elsewhere too)